cloudinit/config/cc_ssh_import_id.py

```diff
@@ -50,7 +50,7 @@
 
 def handle(_name, cfg, cloud, log, args):
 
-    if cfg.get("ssh_import_id") and not subp.which(SSH_IMPORT_ID_BINARY):
+    if not subp.which(SSH_IMPORT_ID_BINARY):
```

Comment:
The reason I second-guessedthis first suggestion, is because now we will always emit warnings from the module on any system that doesn't have ssh-import-id, even if there is no user-data that requests the use of this utility. I don't think that's the behavior we want, we only want to warn and return if we happen to have an ssh-import-id request in user-data.